### ISSUE TITLE
Handling "incompatible existing annotation" error

### ISSUE THREAD

**Issue by ahussain3** [Instagram/MonkeyType#81]
I am seeing the following error:
```
ERROR: Failed applying stub with retype:
Annotation problem in function 'foo': 206:1: incompatible existing return value. 
Expected: 'Tuple[str, datetime]', actual: 'Tuple[sessions.SessionId, DateTime]'
```
Is there a way to address the conflict and progress beyond this error, by doing one of:
* overwrite the existing type with the new generated type
* ignore the generated type and leave the existing type as it, continue applying types to functions where there is no conflict.

As it stands I cannot get to a point where all the "best guess" types have been applied, so I cannot look at a diff and manually fix any problems I see. Instead, monkeytype crashes on one incompatibility and doesn't apply any types.

**Comment by carljm** [comment]
Thanks for the report! Sorry for the slow reply. 

It looks like this might be due to #11, but what I'm not sure of is why we don't just take the existing annotation and prefer it over anything inferred; this is what we're supposed to do. It would help if you can provide a small example code file that reproduces this issue.

**Comment by carljm** [comment]
Fixed with merge of #129
